cms/templates/js/timed-examination-preference-editor.underscore

```diff
@@ -69,5 +69,16 @@
                 <p class='field-message' id='review-rules-description'><%- gettext('Specify any additional rules or rule exceptions that the proctoring review team should enforce when reviewing the videos. For example, you could specify that calculators are allowed.') %></p>
             </ul>
         </div>
+        <div class='timed-exam-options-list-fields is-hidden'>
```

Comment:
@chris-mike I find this UI pretty funky. Two things bother me:

1. The excessive list nesting (even though there is only a single item).
2. The fact that optionally appearing sections always appear at the end, instead of being nested within the exam type that they relate to. I would expect this checkbox to appear within the "Timed" exam section instead of at the end (since we are only supporting it for Timed exams, not Proctored or Practice Proctored exams). Similarly, I think the "Review Rules" text field should appear within the "Proctored" exam section.